triggers/README.md

```diff
@@ -17,14 +29,31 @@ We use Google Cloud Build (GCB) triggers to test the applications.
         trigger.
     *   `[PROJECT_ID]` is the GCP project ID where the trigger will be created.
```

Comment:
*   `[PROJECT_ID]` is the GCP project ID of the project where the trigger will be created.